examples/meteringlab/c2/cassandra-stress.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: dataloader
+  namespace: default
+  labels:
+    app: dataloader
+spec:
+  parallelism: 4
```

Comment:
this will create 4 pods with exactly the same spec (one originating from the job template you provide below), this would use the same secrets, but a connection bundle for each scylla cluster is different. So you need 4 distinct jobs, because their template (volumes) will differ